src/main/java/com/example/alaa/university/domain/University.java

```diff
@@ -0,0 +1,94 @@
+package com.example.alaa.university.domain;
+
+import java.time.Instant;
+import java.util.List;
+
+public class University {
+    private Integer id;
+    private String name;
+    private Address address;
+    private UniversityType universityType;
+    private  String email;
+    private  Double studyCost;
```

Comment:
it is suggested to remove unnecessary spaces. You can use ctrl + alt + l that format the code

Reply:
ok,